packages/web/styles/theme.js

```diff
@@ -0,0 +1,33 @@
+const theme = {
+	colors: {
+		primary: '#EF4136',
```

Comment:
Seria interessante utilizar um único formato para cores, ou tudo em rgba ou tudo em hex. Acho que rgba seria melhor, apesar de já ter visto que HSLA é interessante. https://css-tricks.com/hsl-hsla-is-great-for-programmatic-color-control/

Reply:
Ok. Foi tudo para HSLA.